onnxruntime/python/tools/tensorrt/perf/mem_test/main.cpp

```diff
@@ -0,0 +1,385 @@
+// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
+//
+
+#include <assert.h>
+#include <vector>
+#include <onnxruntime_cxx_api.h>
+#include <tensorrt_provider_factory.h>
+
+std::unique_ptr<OrtTensorRTProviderOptions> get_default_trt_provider_options() {
+  auto tensorrt_options = std::make_unique<OrtTensorRTProviderOptions>();
+  tensorrt_options.get()->device_id = 0;
```

Comment:
don't need get()
just do tensorrt_options->device_id  
same with rest of file.

Reply:
Rewrote the code.